.eslintrc

```diff
@@ -187,10 +188,12 @@
         "extensions/**/test/**/*.js",
         "extensions/**/test-e2e/*.js",
         "ads/**/test/**/*.js",
-        "testing/**/*.js"
+        "testing/**/*.js",
+        "build-system/**/*.js"
```

Comment:
Adding `build-system/` here will cause all the other rules to be disabled as well. Instead, you can simply add `"local/no-destructure-assignment-params": 0,` to this list:

https://github.com/ampproject/amphtml/blob/23201dd241631754eec59a9d27d734d950bb88bb/build-system/.eslintrc#L21-L29

Reply:
done